### DISCUSSION TITLE
Reset Settings Option

### DISCUSSION THREAD

**Discussion by RenKorrum** [baffler/Transparent-Twitch-Chat-Overlay#165] [Ideas]
I had an issue with the app that broke my streamlabs alerts widget, and deleting the settings out of appdata, and reinstalling resolved the issue after configuring the program again. Can we get a button to reset the settings back to default? It'll also help users who accidentally break something (which is probably what happened in my case anyways).

**Comment by baffler** [comment]
This doesn't exactly fix the issue, but you can now at least uninstall the app and it will remove the settings now. With [v0.9.5](https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/tag/v0.9.5)
